ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/policy_maps.schema.yml

```diff
@@ -168,3 +168,22 @@ keys:
                           - "mbytes"
                           - "packets"
                         default: "bytes"
+      copp:
+        type: dict
+        keys:
+          name:
+            type: str
+            description: Name of control-plane poilcy.
+          classes:
+            type: list
+            primary_key: name
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                shape:
+                  type: int
+                  description: Shape value in pps.
```

Comment:
Do we have a min and max?

Reply:
Yes. And we have min/max for bandwidth as well

Reply:
Fixed.